adapters/bidder.go

```diff
@@ -138,6 +139,25 @@ func (r *RequestData) SetBasicAuth(username string, password string) {
 type ExtraRequestInfo struct {
 	PbsEntryPoint              metrics.RequestType
 	GlobalPrivacyControlHeader string
+	currencyConversions        currency.Conversions
```

Comment:
I almost forgot about the `ExtraRequestInfo` struct. This is a great use for it.

Reply:
Agreed. I view this as a more broad "stuff from PBS-Core" object. I suggest we consider renaming it in the future, but not in this PR.